angular.json

```diff
@@ -111,6 +111,7 @@
             "styles": [
               "app/src/assets/fonts/material-icons/material-icons.css",
               "app/src/assets/fonts/muli/muli.css",
```

Comment:
previous font probably needs to be removed?

Reply:
removed unused fonts from angular.json and assets folder.